naming_conventions.md

```diff
@@ -1,88 +1,66 @@
-XML naming conventions
+XML Naming Conventions
 ======================
 
-Basic principle
+Basic Principle
 ---------------
+All resource names are constructed from various components.
 
-All resource names follow a simple convention.
-
-#`<WHAT>_<WHERE>_<DESCRIPTION>_<SIZE>`
-
-Let's first describe every element briefly. After the advantages, we'll
-see how this applies to each resource type.
-
-####`<WHAT>`
-
-Indicate what the resource actually represents, often a standard Android
-view class. Limited options per resource type. (e.g. MainActivity -&gt;
-`activity`)
+#### `<WHAT>`
+Indicate what the resource actually represents. This is often a standard Android view class. Limited 
+options per resource type.
 
```

Comment:
"Limited options per resource type." is unclear. What does this mean? Also it should have a "There are..." in front of it.